.github/workflows/ci.yml

```diff
@@ -104,8 +104,6 @@ jobs:
     with:
       project-name: Component[OpenTelemetry.Exporter.Geneva]
       code-cov-name: Exporter.Geneva
-      os-list: '[ "ubuntu-24.04" ]' # Note: This may be switched to latest once ubuntu-latest has a kernel version >= 6.8.0-1014-azure
-      tfm-list: '[ "net8.0" ]' # Note: Should be able to remove this once the above is using ubuntu-latest
```

Comment:
ubuntu-latest is pointing to a newer version than ubuntu-24.04 now.

* `build-test-exporter-geneva-integration / build-test (ubuntu-24.04, net8.0)` runs on Ubuntu **22.04.1** LTS.
* If not set, the version is newer. For example, `build-test-exporter-influxdb / build-test (ubuntu-latest, net8.0)` runs on Ubuntu **22.04.5** LTS.

Reply:
This was a mistake. `build-test-exporter-geneva-integration / build-test (ubuntu-24.04, net8.0)` must have been running on Ubuntu **24**.04.1 LTS. Because the other version used by the `influxdb` job was 22.04.5, I thought geneva job was **22**.04.**1** which would be older version than `influxdb` job.